Changes to documetation (#11)
CleanMachine1/DollarSkip: README.md

### Manual install

> __You need to have **git** and **make** installed to download the files and compile easier!__

Reviewer: ```suggestion
__To download and compile DollarSkip you need **git** (to download the files) and **make** (to make compilation easier)__
if you don't want to use git and make, you can download the zip from [here](https://github.com/CleanMachine1/DollarSkip/archive/refs/heads/master.zip) and compile `dollarskip.c` with `gcc dollarskip.c -o $`, then move `$` to `/usr/bin`.
```

Author: I personally don't like this however it could be edited.

I found trouble with making the binary from gcc called $, instead I called it temp so it could be moved easier without complications.

I will add my suggestions a little later

Reviewer: it can be:

if you don't want to use git and make, you can download the zip from [here](https://github.com/CleanMachine1/DollarSkip/archive/refs/heads/master.zip) and compile `dollarskip.c` with `gcc dollarskip.c -o temp`, then move `temp` to `/usr/bin/$` with `sudo mv temp /usr/bin/\$`.

Author: That will do however I still don't like the appearance of it physically. So I will put it in a summary.

Author: ```suggestion

<details>
<summary><b>To install DollarSkip without Git or Make</b></summary>

If you don't want to use Git and Make, you can download the zip from [here](https://github.com/CleanMachine1/DollarSkip/archive/refs/heads/master.zip) and compile `dollarskip.c` with `gcc dollarskip.c -o temp`, then move `temp` to `/usr/bin/$` with `sudo mv temp /usr/bin/\$`.
</details>

```

To install, run the following commands in a terminal:

```bash
cd ~/ # Or your place of choice for Git repositories
git clone https://github.com/CleanMachine1/DollarSkip # Or use SSH
cd DollarSkip 
make # This makes the binary

> sudo make install # This moves the binary to /usr/bin

Reviewer: ```suggestion
sudo make install # This moves the binary to /usr/bin and renames it to '$'
```

Author: ```suggestion
sudo make install # This copies the binary to /usr/bin and names it '$'
```

### Manual install

> __To download and compile DollarSkip you need **git** (to download the files) and **make** (to make compilation easier)__

Reviewer 2: ```suggestion
__To download and compile DollarSkip you need **Git** (to download the files) and **Make** (to make compilation easier)__
```
